src/shared/azureagent/customroles.go

```diff
@@ -31,6 +31,15 @@ func (a *Agent) GenerateCustomRoleName(uai armmsi.Identity, scope string) string
 	return agentutils.TruncateHashName(fullName, maxRoleNameLength)
 }
 
+func (a *Agent) ValidateScope(ctx context.Context, scope string) bool {
+	res, err := a.resourceClient.GetByID(ctx, scope, "2022-09-01", nil)
+	if err != nil {
+		return false
```

Comment:
muting errors is generally bad practice, and can hide very bad things. 
Change it to a function returning error (instead of bool), so that `if err != nil { return errors.Wrap(err) };   else if res.GenericResource.ID != nil { return errors.Format(...not found) } ; else return nil`

That's also more in-line with the function name ("Validate" should validate something or fail, while function named "IsScopeExisting" would be expected to return a boolean)